scripts/masker/dma-collection-masker

```diff
@@ -22,9 +22,17 @@ import sys
 import tarfile
 import zipfile as zf
 from collections import namedtuple
+from datetime import datetime
+from enum import Flag, auto
 from pathlib import Path
 from tempfile import NamedTemporaryFile, TemporaryDirectory
-from typing import Dict, List, Optional, Tuple
+
+from faker import Faker
```

Comment:
The original script had no 3rd party dependencies, but I think the benefits you've added with `faker` probably warrant this type of change.

We'll have to write a quick set of instructions for spinning up a venv and installing the requirements (`packaging` and `faker` are the only requirements I can tell).

Reply:
Upon review the only real benefit we got from using `faker` was the hostname generator produced a more human friendly hostname. I've been able to replicate that functionality in our own class so have removed the use of `faker`.

Reply:
The use of packaging is a bit more problematic and I'd like to discuss this.

We could use `distutils.version` but this is deprecated and being replaced by `packaging`, and it will be/is removed in Python 3.12 so I'm not sure on the best path forward on this one.

Reply:
Notes from our call: we agreed to leave `packaging` as a requirement.